packages/react/src/lib/callable.ts

```diff
@@ -1,6 +1,10 @@
+import { Func } from '../types'
+
 // https://jsperf.com/startdust-callable
```

Comment:
we may safely remove this reference as 
- it is pointing to stale document
- we have all the necessary tooling now to measure performance on the Stardust's side

Reply:
Let's remove it 👍